sbnanalysis/CMakeLists.txt

```diff
@@ -23,5 +24,8 @@ file(WRITE ${CMAKE_INSTALL_PREFIX}/bin/setup_sbnanalysis.sh
 "export SBN_ANALYSIS_DIR=${CMAKE_INSTALL_PREFIX}
 export SBN_LIB_DIR=$SBN_ANALYSIS_DIR/lib
 export LD_LIBRARY_PATH=$SBN_LIB_DIR:$LD_LIBRARY_PATH
-export PATH=$SBN_ANALYSIS_DIR/bin:$PATH")
+export PATH=$SBN_ANALYSIS_DIR/bin:$PATH
+export SBN_ANALYSIS_FHICL_DIR=${CMAKE_INSTALL_PREFIX}/fcl
+export FHICL_FILE_PATH=${CMAKE_INSTALL_PREFIX}/fcl:$FHICL_FILE_PATH
```

Comment:
Could you prepend `.:` here, so the CWD stays at the front of the `$FHICL_FILE_PATH`?

Reply:
Fixed